java/src/jmri/jmrix/openlcb/swing/hub/Bundle.properties

```diff
@@ -7,7 +7,7 @@ SendLineTermination     = Send Line Termination
 RequireLineTermination  = Require Line Termination
 
 HubStarted              = {0} {1} started
-IpAddressLine           = Hub ( {0}{1}{2} ) IP Address {3}
+IpAddressLine           = Hub ( {0}{1}{2} ) IP Address {3} {4}
```

Comment:
`IP Address {3} port {4}` might make it more clear here.

Reply:
Agreed, updated.